src/Akka.Streams.Kafka/Stages/Producers/DefaultProducerStage.cs

```diff
@@ -138,7 +131,7 @@ public override void PreStart()
         {
             base.PreStart();
 
-            Producer = _stage.ProducerProvider(HandleProduceError);
+            Producer = _stage.ProducerProvider(null);
```

Comment:
Does `HandleProduceError` not get called anymore?

Reply:
Nope, we don't care about it. everything is handled inside the new message received callback. It is the same thing, the error handler is called before the message is passed to the message received callback.